homework/v.paslavskiy/Hometask 2.1.py

```diff
@@ -0,0 +1,14 @@
+lst = [1, [2, 3], 4, [[6, 7]]]
+
+lst2=[]
+
+for a in lst:
```

Comment:
This is a valid solution to given task but it's nor general nor optimal, because it will fail in case of deeper level of nesting. Ie [1, [2, 3], 4, [[[6, 7],8]]] .

we can talk in next lessons how to improve it.